src/includes/performance/custom-performance-metrics/javascript.mdx

```diff
@@ -1,14 +1,16 @@
-Supported in Sentry's JavaScript SDK version `7.0.0` and above.
+Adding custom metrics is supported in Sentry's JavaScript SDK version `7.0.0` and above.
 
 ```javascript
-const transaction = Sentry.getCurrentHub().getScope().getTransaction();
+const transaction = Sentry.getCurrentHub()
+  .getScope()
+  .getTransaction();
 
 // Record amount of memory used
-transaction.setMeasurement('memoryUsed', 123, 'byte');
+transaction.setMeasurement("memoryUsed", 123, "byte");
```

Comment:
I'm curious if this is auto-formatting or a conscious choice. (In the JS SDK, our standard is single quotes. Makes no difference to the computer, so neither one is "right," but it means that any code that we copy in from our end is going to have single quotes.)

Reply:
Definitely auto-formatting. I don't love the idea of it being inconsistent (though I'm sure I've already caused this with my linter), and I think y'all should be the authority on which types of quotes we use, so I'm gonna kill these edits.